<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="comboBox_scale">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1uV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10uV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>25uV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>50uV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100uV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>250uV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>500uV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>1mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2.5mV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>100mV</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="checkBox_car">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>97</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>CAR Filter</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time (s)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Scale</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_bandpass">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Bandpass Filter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bp">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>17</y>
      <width>99</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply BP</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_hp">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>17</y>
      <width>69</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>0.100000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_lp">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>17</y>
      <width>69</width>
      <height>27</height>
     </rect>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1.000000000000000</double>
    </property>
    <property name="value">
     <double>50.000000000000000</double>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showTID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>161</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show TID events</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showLPT">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show LPT events</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_showKey">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Key events</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>-1</y>
      <width>21</width>
      <height>133</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>45</y>
      <width>421</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>57</y>
      <width>21</width>
      <height>75</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_stoprec">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>95</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop REC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_rec">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>REC</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_recFilename">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>60</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_channels">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>131</y>
      <width>525</width>
      <height>503</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>529</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
